src/components/tools/search.js

```diff
@@ -237,6 +237,14 @@ const correct = (person) => {
             }
             person.death = undefined;
         }
+    } else if ( person.death && (person.death.age !== null) && (person.death.age >=0) && (person.death.age < 18) ) {
```

Comment:
Il me semble que les deux conditions >= 0 et < 18 sont en doublon.
```suggestion
    } else if ( person.death && (person.death.age !== null) && (person.death.age < 18) ) {
```